src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Debug/DebugProfileEnumValuesGenerator.cs

```diff
@@ -26,19 +28,21 @@ internal class DebugProfileEnumValuesGenerator : IDynamicEnumValuesGenerator
         /// <summary>
         /// Create a new instance of the class.
         /// </summary>
-        internal DebugProfileEnumValuesGenerator(ILaunchSettingsProvider profileProvider)
+        internal DebugProfileEnumValuesGenerator(
+            ILaunchSettingsProvider profileProvider,
+            IProjectThreadingService threadingService)
         {
```

Comment:
I see you dereference your `threadingService` parameter without validating that it isn't null. Consider adding:
```csharp
Requires.NotNull(threadingService, nameof(threadingService));
```